web/src/components/base/Checkbox.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+
+const StyledCheckbox = styled.input`
+  cursor: pointer;
+  accent-color: ${props => props.theme.colors.themeColor};
+`
+
+type CheckboxProps = {
+  checked: boolean
+  setChecked: (checked: boolean) => void
+}
+
+const Checkbox = ({ checked, setChecked }: CheckboxProps): ReactElement => (
+  <StyledCheckbox type='checkbox' checked={checked} onChange={() => setChecked(!checked)} />
```

Comment:
A good practise is to connect the checkbox with a `<label>` and to use the `for` value to connect. Then you can click the checkbox and the label to check which is better ux imo. Additionally this is good a11y because the screen reader know for what the checkbox is
So maybe it would be nice to optionally pass a text which will be rendered in a label tag